docs/installation/installation-framework.md

```diff
@@ -49,25 +49,21 @@ see the above section on installing `iTHOR`.
 To run experiments using Habitat please use our docker image using the following command:
 
 ```bash
-docker pull klemenkotar/allenact:latest
+docker pull allenact/allenact:latest
 ```
 
 This container includes the 0.1.0 release of `allenact`, the 0.1.5 release of `habitat` as well
 as the `Gibson` point navigation dataset. This dataset consists of a set of start and goal positions provided by habitat.
 You then need to launch the container and attach into it:
 
 ```bash
-docker run --runtime=nvidia -it klemenkotar/allenact
+docker run --runtime=nvidia -it allenact/allenact
```

Comment:
While we're here, what happens if I don't have a GPU? Do I just remove `--runtime=nvidia` and everything works? If not, we should definitely have some way of enabling cpu-only training.

Reply:
Yes, that is correct we just remove `--runtime=nvidia` but I think that training habitat without a GPU is pretty much pointless. Even evaluations take a very long time. Nonetheless it is worth mentioning.